src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -36,12 +36,31 @@
 
     private static final long serialVersionUID = 1;
 
-    public PullRequestSCMHead(int number) {
+    private String user;
```

Comment:
A `user` entity is completly [different](https://developer.github.com/v3/users/#get-a-single-user).